.github/workflows/osrm-backend.yml

```diff
@@ -649,37 +649,17 @@ jobs:
       GITHUB_REPOSITORY: ${{ github.repository }}
       RUN_BIG_BENCHMARK: ${{ contains(github.event.pull_request.labels.*.name, 'Performance') }}
     steps: 
-      - name: Enable data.osm.pbf cache
```

Comment:
It is not needed anymore - we don't clean cache on self-hosted runner, so job runs even faster without these caching actions...